### ISSUE TITLE
Initial feedback on taxonomy browser

### ISSUE THREAD

**Issue by kcranston** [OpenTreeOfLife/opentree#828]
- [ ] make taxon name much more prominent on page (hard to find now)
- [ ] reverse order of breadcrumb / lineage: `life > child > child`
- [ ] different styles for Taxon, Synonyms, Lineage, Children
- [ ] remove asterisks and replace with separate heading “Suppressed from synthetic tree”
- [ ] header on page with “Open Tree of Life Taxonomy”, with version number and links to download page and github repo
- [ ] check sort order of children - seems inconsistent right now
- [ ] separate style for internal and external links

**Comment by jar398** [comment]
How about removing the bullets from the children list.  They are just clutter

**Comment by jimallman** [comment]
> How about removing the bullets from the children list. They are just clutter

I've addressed this and several other points, including adding `H3` headers for organization, in the latest commit above. Here's a screenshot of the latest:

<img width="820" alt="current-tags" src="https://cloud.githubusercontent.com/assets/446375/11985640/e953e85a-a994-11e5-8daf-05daefde2d79.png">


... and a modified version with more prominent flags (we should talk about which elements deserve greater emphasis):

<img width="909" alt="stronger-tags" src="https://cloud.githubusercontent.com/assets/446375/11985639/e2e9b468-a994-11e5-9795-b0cc276146af.png">

**Comment by jar398** [comment]
Great.

I was also thinking maybe it might be nice to make the names line up...
either by putting the rank after the name, or by ​using a 2-column table
with right-justified ranks in one column and everything else in the other.

I have a feeling that the better this thing looks, the more changes we'll
want to make to it!

**Comment by mtholder** [comment]
I think that this is looking great and that this will be a *great* addition to the open tree web services. Thanks, @jar398, @jimallman, and @kcranston !

**Comment by kcranston** [comment]
Agreed - I think this is ready for v 1.0 release. I suppose we need an issue for making the links in the web application. Look! Here's one: #830
